cost-analyzer/templates/cost-analyzer-cluser-role-template.yaml

```diff
@@ -1,3 +1,19 @@
+# Log Access restricted to kubecost release namespace
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Lgtm, but can we also potentially add a comment on why we are getting these? Maybe something like this:

```
# Logs Access used for Kubecost bug report feature -- can be removed if desired 
```

Reply:
Going to move to a values.yaml config so we can document better and cleanly control enabled/disabled.